Makefile

```diff
@@ -34,7 +34,7 @@ applyTemplate:
 	oc process -f $(WHAT) | oc apply -f -
 .PHONY: applyTemplate
 
-postsubmit-update: prow-services origin-release libpod prow-monitoring build-dashboards-validation-image image-mirror-setup knative-image-mirror-setup tekton-image-mirror-setup kubefed-image-mirror-setup toolchain-image-mirror-setup cincinnati
+postsubmit-update: prow-services origin-release libpod prow-monitoring build-dashboards-validation-image image-mirror-setup knative-image-mirror-setup tekton-image-mirror-setup kubefed-image-mirror-setup toolchain-image-mirror-setup codeready-toolchain-image-mirror-setup cincinnati
```

Comment:
Is `codeready-toolchain` something different than `toolchain` that already exists there?

Reply:
yes, it's different. I would like to put all new codeready-toolchain related operators under `codeready-toolchain`.